src/test/java/plugins/SubversionPluginTest.java

```diff
@@ -49,7 +54,8 @@ public void run_basic_subversion_build() throws SubversionPluginTestException {
         f.addShellStep("test -d .svn");
         f.save();
 
-        f.startBuild().shouldSucceed().shouldContainsConsoleOutput("test -d .svn");
+        Build b = f.startBuild().shouldSucceed();
+        assertThat(b.getConsole(), Matchers.containsRegexp("test -d .svn", Pattern.MULTILINE));
```

Comment:
@jglick The deprecated method suggested the use of the <code>containsRegexp</code> method in the Javadoc, but you are absolutely right,in this case it may match other strings.

I was not able to find the method <code>Matcher.containsString</code>, so I have used the <code>String.contains</code> method instead.